internal/codegen/tpl_module.go

```diff
@@ -55,11 +55,8 @@ type TplModule struct {
 //
 //	{{ module.Export "HelloWorld" }}
 func (tm *TplModule) Export(name string) (string, error) {
-	tm.s.moduleCaller.mu.Lock()
-	defer tm.s.moduleCaller.mu.Unlock()
-
 	// We only allow functions to be exported in the first pass.
-	if !tm.s.isFirstPass {
+	if !tm.s.isRenderStage {
```

Comment:
this is backwards now, right?

Reply:
Nah, but the confusion makes it seem like it is :D